docs/challenges/language-support/typescript.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: "Typescript"
+description: "Learn how to solve CodeCrafters challenges in Typescript"
+---
+
+# Local development
+
+To run the code for your challenge locally, you'll need `bun` [installed](https://bun.sh/docs/installation). Our test runners use Bun v1.1 (as of May 2024).
+
+The script for your challenge (like `./your_grep.sh`) will automatically compile your code using `bun` before executing it. It'll
```

Comment:
Bun doesn't "compile" code - needs re-wording. Look at Python for an example

Reply:
Woops, sorry, missed this